CHANGELOG.md

```diff
@@ -1,7 +1,76 @@
 # Ezra Bible App - Changelog
 All notable changes to Ezra Bible App are documented in this file.
 
-### [1.0.0 - 2021-04-24]
+## [1.1.0 - 2021-06-26]
+
+### Features
+
+* Desktop app: Added emoji picker to input fields (Create/rename tag, notes). ([#166](i166))
+* Added a locale select box in the options menu that can be used to instantly switch to a different user interface language. ([#203](i203))
+* Added functionality for opening and navigating book chapters individually. ([#283](i283))
+
+### Enhancements
+
+* Enhanced layout of current tab search function. ([#165](i165))
+* Made dictionary component more responsive. ([#240](i240))
+* Fixed text rendering issues with New English Translation. ([#237](i237))
+* Disable Bible translation in translation select box if it does not contain the book that is opened in the current tab. ([#274](i274))
+* Tag statistics - hide tags that have 0 occurrences. ([#324](i324))
+* Added online help that gives an overview for all keyboard shortcuts (New tab in info popup dialog). ([#198](i198))
+* Android: Localized startup messages. ([#323](i323))
+
```

Comment:
I just realized this after approving the review.

Maybe add
* Introduced web components. Refactored display options as web components #247

Reply:
Ok. This would be an enabler. Would you mind pushing that update to this feature branch?